docs/DeveloperGuide.md

```diff
@@ -320,14 +321,12 @@ Step 1. The user executes a `view benson meier` command which puts the `Person`
 Step 2. The user executes an `edit s/1 Mon 08:30` command which edits the index 0 of the `SessionList` displayed on the GUI. This creates a new Session instance with the string input of the format `EEE HH:mm`.
 		The `Session` instances in the `SessionList` of the `Person` is immediately sorted using the `Session` class `compareTo` implementation.
 
-Step 3. The user executes a `list` command which puts the Pupilist application into list view, displaying multiple Persons to the user.
-
-Step 4. The user executes an `add 1 s/ Tue 09:00` command which creates a new instance of `Session` with the string input of the format `EEE HH:mm` and adds it to the `SessionList` of the `Person` of the first Person Card shown in the GUI.
+Step 3. The user executes an `session 1 s/ Tue 09:00` command which creates a new instance of `Session` with the string input of the format `EEE HH:mm` and adds it to the `SessionList` of the `Person` of the first Person Card shown in the GUI.
 	The `Session` instances in the modified `SessionList` of the `Person` is immediately sorted using the `Session` class `compareTo` implementation.
 	
-Step 5. The user executes a `view Alice Pauline` command which puts the `Person` with the name Alice Pauline in view mode.
+Step 4. The user executes a `view Alice Pauline` command which puts the `Person` with the name Alice Pauline in view mode.
 
-Step 6. The user executes a `remove s/3` command which removes the instance of `Session` in index 2 of the `SessionList` of the `Person` displayed in the GUI in view mode. The Session 
+Step 5. The user executes a `remove s/3` command which removes the instance of `Session` in actual index 2 of the `SessionList` of the `Person` displayed in the GUI in view mode. The Session 
```

Comment:
think this index part is abit confusing, could specify that index 3 is list displayed to the user but internal sessionlist uses a zero based index? (same with the above where you say edit index 0)

Reply:
okkie will make the change!